File: app/src/main/java/com/robotnec/budget/core/mvp/view/CategoryOverviewView.java
package com.robotnec.budget.core.mvp.view;

import com.robotnec.budget.core.service.aggregation.impl.TransactionAggregation;

/**
 * @author zak <zak@swingpulse.com>
 */
public interface CategoryOverviewView extends View {
    void displayCategoryTitle(String title);

    void displayCategoryTransactions(TransactionAggregation transactionAggregation);
}

File: app/src/main/java/com/robotnec/budget/core/mvp/view/CategoryOverviewView.kt
package com.robotnec.budget.core.mvp.view

import com.robotnec.budget.core.service.aggregation.impl.TransactionAggregation

/**
 * @author zak zak@swingpulse.com>
 */
interface CategoryOverviewView : View {
    fun displayCategoryTitle(title: String)

    fun displayCategoryTransactions(transactionAggregation: TransactionAggregation)
}
